src/main/java/com/microsoft/sqlserver/jdbc/Figure.java

```diff
@@ -0,0 +1,48 @@
+/*
+ * Microsoft JDBC Driver for SQL Server Copyright(c) Microsoft Corporation All rights reserved. This program is made
+ * available under the terms of the MIT License. See the LICENSE file in the project root for more information.
+ */
+
+package com.microsoft.sqlserver.jdbc;
+
+/**
+ * Represents the internal makings of a Figure.
+ *
+ */
+public class Figure {

```

Comment:
Do we want to rename these classes to something like `SQLServerFigure` to maintain consistency.

Reply:
These classes don't seem to belong to microsoft.sql package. I would suggest to create a `com.microsoft.sqlserver.jdbc.spatialdatatypes` package and move **all** related classes under that package. See `com.microsoft.sqlserver.jdbc.dataclassification` for reference.